src/config/print/index.md

```diff
@@ -66,7 +67,8 @@ install the `avahi` package and enable the `avahi-daemon` service.
 
 To configure the printer using the CUPS web interface, navigate to
 <http://localhost:631> in a browser. Under the "Administration" tab, select
-"Printers > Add Printer".
+"Printers > Add Printer". Log in using an account that is in the `lpadmin`
```

Comment:
This might read more clearly if it was at the end of the sentence about logging into CUPS? Not a big deal really.

Reply:
Well, if I follow these instructions, I get the login dialog after clicking the 'Add Printer' button. I rephrased it a bit to improve readability.